src/widgets/study-path/ui/study-path.scss

```diff
@@ -14,7 +14,7 @@
       }
     }
 
-    & > .paragraph {
+    & > .study-path-paragraph {
```

Comment:
Please, remove this styles - it's duplicate our `Paragraph medium` styles

Reply:
This is a specific behavior, it only occurs here and if it is replaced with the standard behavior, the current implementation will differ from the layout...

Reply:
[Resolved](https://github.com/rolling-scopes/site/pull/475/commits/c2a047a0dedc9dbacb91d4e2a6adb6117bb32111).